lib.php

```diff
@@ -2897,6 +2897,24 @@ function facetoface_cm_info_view(cm_info $coursemodule) {
                     $futuresessions[] = $sessionobject;
                 }
 
+                // Check if custom fields exist, and add to sessionobject if setting is enabled.
+                $visiblefieldcolumn = get_config('facetoface', 'column');
+                if ($visiblefieldcolumn) {
+                    // Get field title.
+                    $fieldname = $DB->get_field('facetoface_session_field', 'name', [
+                        'id' => $visiblefieldcolumn,
+                    ]);
+                    // Get field value for the session.
+                    $fieldvalue = $DB->get_field('facetoface_session_data', 'data', [
+                        'fieldid' => $visiblefieldcolumn,
+                        'sessionid' => $session->id
+                    ]);
+                    $sessionobject->field = (object)[
```

Comment:
`$sessionobject->field` is a bit vague. Maybe make it at least `$sessionobject->customfield`?